src/owncloud/setup_owncloud

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
+############################################
+# This script is called by /bin/envvars
+############################################
+#
+. $SNAP/setup/filesystem.sh
```

Comment:
There is no setup script panned for Snappy, so this has to be included in every daemon script